app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -300,9 +300,16 @@ type VirtualCartItem implements CartItemInterface @doc(description: "Virtual Car
 interface CartItemInterface @typeResolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\CartItemTypeResolver") {
     id: String!
     quantity: Float!
+    prices: CartItemPrices @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\CartItemPrices")
```

Comment:
Current implementation is based on the assumption that prices can be extracted from the `product` field. If this assumption is incorrect, please explain.

Reply:
I've worked on a number of sites with external integrations powering cart pricing. In these stores, most B2B with contract pricing, there can be a disconnect between the product price and the cart item price.